### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T848278)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# File Manager - How to use it in ASP.NET Core Razor Pages
+# File Manager for ASP.NET Core - How to use file manager in ASP.NET Core Razor Pages
 
-This example shows DevExtreme FileManager, which is bound to a default file sytem provider. The [page handler methods](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-3.1&tabs=visual-studio#multiple-handlers-per-page) work as backend here. 
+This example demonstrates DevExtreme [FileManager](https://docs.devexpress.com/AspNetCore/401320/devextreme-based-controls/controls/file-manager) control bound to a default file sytem provider. The [page handler methods](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-3.1&tabs=visual-studio#multiple-handlers-per-page) work as backend.
```
Comment (albertov05) [inline README.md]: 1. Please update the first sentence as follows:

"This example demonstrates **the** DevExtreme [FileManager](https://docs.devexpress.com/AspNetCore/401320/devextreme-based-controls/controls/file-manager) control bound to a default file **system** provider."

2. By "work as backend" do you mean "work on the backend"?

File: README.md [hunk 2]
```diff
@@ -1,42 +1,40 @@
...
+    ```
 
-2) Add File Manager control to your Razor Page and use the Remote provider.
-```cs
-@(Html.DevExtreme().FileManager()
-    .FileSystemProvider(provider =>
-        provider.Remote()
-        .Url(Url.Page("Index", "Documents")))
-```
+3. File Manager uses both `GET` and `POST` requests, so it's necessary to create handlers for both request types.
```
Comment (albertov05) [inline README.md]:
```suggestion
3. File Manager uses both `GET` and `POST` requests, so it is necessary to create handlers for both request types.
```

File: README.md [hunk 3]
```diff
@@ -1,42 +1,40 @@
...
-
-```cs
- return command == FileSystemCommand.Download ? (IActionResult)result : new JsonResult(result);
-```
-5) Disable AntiForgeryToken in the Razor Page with File Manager via [IgnoreAntiforgeryTokenAttribute](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.ignoreantiforgerytokenattribute?view=aspnetcore-3.1). 
+    ```cs
+    public IActionResult OnGetDocuments(FileSystemCommand c, string a) => ProcessFileApiRequest(c, a);
+    public IActionResult OnPostDocuments(FileSystemCommand c, string a) => ProcessFileApiRequest(c, a);
+    ```
+4. Process requests with [PhysicalFileSystemProvider](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.FileManagement.PhysicalFileSystemProvider) like it is demonstrated in the following demo: [Physical File System](https://demos.devexpress.com/ASPNetCore/Demo/FileManager/BindingToFileSystem/).
```
Comment (albertov05) [inline README.md]:
```suggestion
4. Process requests with [PhysicalFileSystemProvider](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.FileManagement.PhysicalFileSystemProvider) as demonstrated in the following demo: [Physical File System](https://demos.devexpress.com/ASPNetCore/Demo/FileManager/BindingToFileSystem/).
```

File: README.md [hunk 4]
```diff
@@ -1,42 +1,40 @@
...
+  
+    All data operations except **Download** return JSON data. Return the result in the handler as follows:
+    
+    ```cs
+    return command == FileSystemCommand.Download ? (IActionResult)result : new JsonResult(result);
+    ```
+5. Use the [IgnoreAntiforgeryTokenAttribute](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.ignoreantiforgerytokenattribute?view=aspnetcore-3.1) object to disable `AntiForgeryToken` in the Razor Page with File Manager.
```
Comment (albertov05) [inline README.md]:
```suggestion
5. Use the [IgnoreAntiforgeryTokenAttribute](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.ignoreantiforgerytokenattribute?view=aspnetcore-3.1) object to disable `AntiForgeryToken` on the Razor Page with File Manager.
```

File: README.md (hunk 1 above)
Comment (LanaDX) [reply README.md]:
```suggestion
This example demonstrates the DevExtreme [FileManager](https://docs.devexpress.com/AspNetCore/401320/devextreme-based-controls/controls/file-manager) control bound to a default file system provider. The [page handler methods](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-3.1&tabs=visual-studio#multiple-handlers-per-page) work on the backend. 

```

**Pull request merged**